core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js

```diff
@@ -111,4 +111,223 @@ function initLogPage(params, logLen, start, end, totLogLen, defaultLen) {
     disableMoreButton();
   }
-}
\ No newline at end of file
+}
+
+function updateBasicMemoryStats(statsData){
+
+  if(statsData.isLocator){
+    return;
+  }
+
+  var currHeapStoragePoolUsed = convertSizeToHumanReadable(statsData.heapStoragePoolUsed);
+  var currHeapStoragePoolSize = convertSizeToHumanReadable(statsData.heapStoragePoolSize);
+  var currHeapExecutionPoolUsed = convertSizeToHumanReadable(statsData.heapExecutionPoolUsed);
+  var currHeapExecutionPoolSize = convertSizeToHumanReadable(statsData.heapExecutionPoolSize);
+  var currHeapMemoryUsed = convertSizeToHumanReadable(statsData.heapMemoryUsed);
+  var currHeapMemorySize = convertSizeToHumanReadable(statsData.heapMemorySize);
+
+  var currOffHeapStoragePoolUsed = convertSizeToHumanReadable(statsData.offHeapStoragePoolUsed);
+  var currOffHeapStoragePoolSize = convertSizeToHumanReadable(statsData.offHeapStoragePoolSize);
+  var currOffHeapExecutionPoolUsed = convertSizeToHumanReadable(statsData.offHeapExecutionPoolUsed);
+  var currOffHeapExecutionPoolSize = convertSizeToHumanReadable(statsData.offHeapExecutionPoolSize);
+  var currOffHeapMemoryUsed = convertSizeToHumanReadable(statsData.offHeapMemoryUsed);
+  var currOffHeapMemorySize = convertSizeToHumanReadable(statsData.offHeapMemorySize);
+
+  $("#currHeapStoragePool").text(
+    currHeapStoragePoolUsed[0] + " " + currHeapStoragePoolUsed[1] + " / "
+      + currHeapStoragePoolSize[0] + " " + currHeapStoragePoolSize[1] );
+  $("#currHeapExecutionPool").text(
+    currHeapExecutionPoolUsed[0] + " " + currHeapExecutionPoolUsed[1] + " / "
+      + currHeapExecutionPoolSize[0] + " " + currHeapExecutionPoolSize[1]);
+  $("#currHeapMemory").text(
+    currHeapMemoryUsed[0] + " " + currHeapMemoryUsed[1] + " / "
+      + currHeapMemorySize[0] + " " + currHeapMemorySize[1]);
+  $("#currOffHeapStoragePool").text(
+    currOffHeapStoragePoolUsed[0] + " " + currOffHeapStoragePoolUsed[1] + " / "
+      + currOffHeapStoragePoolSize[0] + " " + currOffHeapStoragePoolSize[1]);
+  $("#currOffHeapExecutionPool").text(
+    currOffHeapExecutionPoolUsed[0] + " " + currOffHeapExecutionPoolUsed[1] + " / "
+      + currOffHeapExecutionPoolSize[0] + " " + currOffHeapExecutionPoolSize[1]);
+  $("#currOffHeapMemory").text(
+    currOffHeapMemoryUsed[0] + " " + currOffHeapMemoryUsed[1] + " / "
+      + currOffHeapMemorySize[0] + " " + currOffHeapMemorySize[1]);
+}
+
+function updateUsageCharts(statsData){
+  var cpuChartData = new google.visualization.DataTable();
+  cpuChartData.addColumn('datetime', 'Time of Day');
+  cpuChartData.addColumn('number', 'CPU');
+
+  var heapChartData = new google.visualization.DataTable();
+  heapChartData.addColumn('datetime', 'Time of Day');
+  heapChartData.addColumn('number', 'JVM');
+  heapChartData.addColumn('number', 'Storage');
+  heapChartData.addColumn('number', 'Execution');
+
+  var offHeapChartData = new google.visualization.DataTable();
+  offHeapChartData.addColumn('datetime', 'Time of Day');
```

Comment:
a header OFF-HEAP not required here?

Reply:
Header is specified in chart configuration options.